app/javascript/utilities/animatedImageUtils.jsx

```diff
@@ -16,13 +16,22 @@ export const initializePausableAnimatedImages = (animatedImages = []) => {
   if (animatedImages.length > 0) {
     const freezeframes = [];
 
-    // Remove the surrounding links for the image, so it can be clicked to play/pause
-    for (const image of animatedImages) {
-      image.closest('a').outerHTML = image.outerHTML;
+    for (let i = 0; i < animatedImages.length; i++) {
+      const image = animatedImages[i];
+      // Give the image an ID so that we can uniquely target it in freezeframe
+      image.setAttribute('id', `animated-${i}`);
+
+      // Our markdown parser allows use of the <center> element in posts, which could conflict with CSS we apply to center the freezeframe container
+      // Centering in this way makes sure the position is correct regardless of whether the user has used their own <center> tag
+      const center = document.createElement('center');
```

Comment:
A potential issue is that the GIF could shift on page load since this is done client-side, but in the demo and when I tested this, I did not notice any shifting.

Reply:
I rethought this and we don't need a JS solution for this at all aside from what's in the original implementation.

```css
.ff-frame {
  // center
}

center > .ff-frame {
 // don't center
}
```

Here we're not manipulating the DOM at all. CSS specificity ftw! 😎